app/controllers/circle/members_controller.rb

```diff
@@ -56,4 +57,27 @@ def active_users
       .order('last_name asc')
   end
 
+  def organizers
```

Comment:
It's totally correct to combine the two actions/variations into one and make a private method to couple all the low-level finding stuff. I'd go one step further though, and not leave the low-level model finding/sorting/munging code in the controller; that's not it's responsibility in MVC. I think with the gems & architecture we have in place, a mutation would be the correct spot (despite the name implying otherwise), I'd call it Circle::FindMembers, or similar. Having a whole class for this finding task will allow you to create a few smaller methods to break down the task, and the result should be easier to understand.